_posts/2018-06-13-hackillinois-2019-mission.markdown

```diff
@@ -1,43 +1,38 @@
+HackIllinois 2018 also saw many engaging workshops and lightning talks, teaching everything from Unity development to making memes. Our sponsors and students always do a great job in offering supplemental knowledge and tools that you may not otherwise have had the opportunity to experience. Our sponsors are some of the biggest, best, and most cutting-edge players in the tech industry, and the value they provide to the event and the attendees cannot be overstated. Being able to take a break from your project is something all of our attendees should take advantage of, and we'll continue to develop these aspects of the event for the years to come.
 
-2018 also saw some very engaging workshops and lightning talks, teaching everything from Unity development to making memes. Our sponsors and students always do a great job in offering supplemental knowledge and tools that you may not otherwise have had the opportunity to experience. Our sponsors are some of the biggest, best, and most cutting-edge in the tech industry, and the value they provide to the event and the attendees cannot be overstated. Being able to take a break from your project is something all of our attendees should feel comfortable doing, and we’ll continue to develop these events for the years to come.
+# Something New
 
+There are only 8 months, 2 days, and 4 hours until check-in, but who's counting? HackIllinois 2019 will be the sixth time Illinois students organize what has become the premier open source event available to undergraduates. While writing this statement, we wanted to explore and emphasize the varied roles that HackIllinois takes on. Our mission this year:
 
-# Something New:
+>**To create an environment conducive to education, collaboration, and mentorship within the Open Source Community**
 
-There are only 8 months, 2 days, and 4 hours until check-in, but who’s counting? HackIllinois 2019 will be the 6th time Illinois students organize what has become the premier open source event available to undergraduates. While writing this statement, we wanted to explore and emphasize the varied roles that HackIllinois takes on. We have established our mission this year:
+We _create an enviroment_  in that our event is not defined by a simple "code for a prize" model. We plan to pack the entire event with content for attendees, from Illinois and over a dozen other universities, so that they can have a unique experience throughout the weekend. HackIllinois carries development, social, and networking opportunities, and these establish the three pillars -- _education, collaboration, and mentorship_ -- of our event. The theme this year follows the trail that has been blazed by our past directors and staff, but we wanted to assert our post _within the Open Source community_. We fully believe that our event betters the Open Source Community: by bringing together new students, continuing development on projects we believe in, and showcasing the talent pool that we have to offer.
```

Comment:
confused by the wording of "wanted to assert our post"

Reply:
I initially had 'established' but figured it should be something that was equivalent to 'emphasize'